packages/integrations/src/Namada.ts

```diff
@@ -56,17 +55,4 @@ export default class Namada implements Integration<Account, Signer> {
   public signer(): Signer | undefined {
     return this._namada?.getSigner();
   }
-
-  public async addTxWasmHashes(
```

Comment:
Very minor thing, but these are still present in the `types` docs.

Reply:
Thanks! Updated